charts/kube-state-metrics/templates/psp-clusterrolebinding.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if and .Values.podSecurityPolicy.enabled .Values.rbac.create -}}
+{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```

Comment:
This could be a single line:

```
{{- if and .Values.podSecurityPolicy.enabled .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```